README.md

```diff
@@ -18,6 +18,25 @@ Install the dependencies and start a local instance using the following script:
 yarn start
 ```
 
+## Deploying the UI
+
+1. Change `index.html` to point to `dist/index.js` (instead of `example.js`) and upload to s3 bucket (or other file hosting service).
```

Comment:
This seems a little too detailed and oriented towards deploying to AWS S3. I think it would be sufficient to just have this section be about how to load an index.html file with the built assets.

Reply:
I don't really see what the harm is in including good instructions on how this could actually be used in the real world with AWS.

Reply:
Because:

1. It's code that needs to be maintained over time. 
2. I don't believe that TriMet uses AWS to host these files.
3. It adds unnecessary instructions to the README that will probably be not useful people who don't use AWS.
4. Deploying like this should be relatively common knowledge for experienced developers and they may even use slightly different or even more advanced AWS settings if they use AWS.